flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/restart/FixedDelayRestartStrategy.java

```diff
@@ -108,22 +87,7 @@ public static FixedDelayRestartStrategyFactory createFactory(Configuration confi
 			timeoutString
 		);
 
-		long delay;
-
-		try {
-			delay = Duration.apply(delayString).toMillis();
-		} catch (NumberFormatException nfe) {
-			if (delayString.equals(timeoutString)) {
-				throw new Exception("Invalid config value for " +
-					ConfigConstants.AKKA_WATCH_HEARTBEAT_PAUSE + ": " + timeoutString +
-					". Value must be a valid duration (such as '10 s' or '1 min')");
-			} else {
-				throw new Exception("Invalid config value for " +
-					ConfigConstants.EXECUTION_RETRY_DELAY_KEY + ": " + delayString +
-					". Value must be a valid duration (such as '100 milli' or '10 s')");
-			}
-		}
-
```

Comment:
No we cannot remove this exception handling here, because this method and the code in `RestartStrategyFactory` are two different code paths. So please reintroduce the exception handling.